### PR TITLE
Run code formatter on lib/elixir/test/elixir/kernel/binary_test.exs

### PR DESCRIPTION
**Pull request by fabrik42**
Please let me know if I need to change something. :)

### CODE REVIEW

**Comment by fabrik42** [comment]
It seems like Travis is not happy with [this change](https://github.com/elixir-lang/elixir/pull/6833/files#diff-009c2e38ffccea77b9eee00fdd62a238R138).

However, as far as I can see, this is valid Elixir syntax. It works in my local `iex` and `make` test also shows on error. What should I do?

File: lib/elixir/test/elixir/kernel/binary_test.exs [hunk 1]
```diff
@@ -129,7 +135,7 @@ bar \"""
 
   test "literal" do
     assert <<106, 111, 115, 195, 169>> == <<"josé">>
-    assert <<106, 111, 115, 195, 169>> == <<"#{:"josé"}">>
+    assert <<106, 111, 115, 195, 169>> == <<"#{:josé}">>
```
Comment (josevalim) [inline lib/elixir/test/elixir/kernel/binary_test.exs]: Please write it as: `#{String.to_atom("josé")}`.

**Comment by josevalim** [comment]
@fabrik42 I have left a comment on how it should be fixed, great catch!

**Comment by fabrik42** [comment]
@josevalim Thanks, I updated the test. :)

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**